rosdep/base.yaml

```diff
@@ -602,6 +602,17 @@ dnsmasq:
   fedora: [dnsmasq]
   gentoo: [net-dns/dnsmasq]
   ubuntu: [dnsmasq]
+docker:
+  debian: [docker]
+  fedora: [docker]
```

Comment:
The `docker` packages in Debian and Ubuntu refer to something entirely different from the `docker` package in Fedora. Fedora's `docker` package belongs with the `docker.io` keys for Debian and Ubuntu below.